crates/turborepo-api-client/src/lib.rs

```diff
@@ -536,6 +537,73 @@ impl APIAuth {
     }
 }
 
+// Anon Client
+
+#[async_trait]
+pub trait AnonClient {
```

Comment:
Do we need this trait? It seems a little unnecessary for a single function. Also, we can probably nix the `async_trait` part

Reply:
Good call, done!